docs/Adding-Software-to-Homebrew.md

```diff
@@ -186,6 +237,24 @@ Other commonly used stanzas are:
 
 Additional [`artifact` stanzas](Cask-Cookbook.md#at-least-one-artifact-stanza-is-also-required) may be needed for special use cases. Even more special-use stanzas are listed at [Optional Stanzas](Cask-Cookbook.md#optional-stanzas).
 
+#### Generating a token for the cask
+
+The cask **token** is the mnemonic string people will use to interact with the cask via `brew install`, etc. The name of the cask **file** is simply the token with the extension `.rb` appended.
+
+The easiest way to generate a token for a cask is to run `generate_cask_token`:
+
+```bash
+$(brew --repository homebrew/cask)/developer/bin/generate_cask_token "/full/path/to/new/software.app"
+```
+
+If the software you wish to create a cask for is not installed, or does not have an associated App bundle, just give the full proper name of the software instead of a pathname:
+
+```bash
+$(brew --repository homebrew/cask)/developer/bin/generate_cask_token "Google Chrome"
+```
+
+If the `generate_cask_token` script does not work for you, see [Cask Token Details](#cask-token-details).
+
 #### Cask token details
```

Comment:
This section should definitely be moved to the Cask Cookbook; maybe still linked to as a 1-liner here if super relevant.

There is an existing section in the Cask Cookbook for this sort of thing; and this rule adds relevant nuance that is currently ambiguous there (assuming this is accurate)